services/meta/parameters.md

```diff
@@ -0,0 +1,149 @@
+# Parameters Service
+
+Parameters service allows configuration of a device.
+
+## Service name
+
+`parameters`
+
+## Interfaces
+
+| Type | Interface                 | Value type | Description                                                                                                  |
+|------|---------------------------|------------|--------------------------------------------------------------------------------------------------------------|
+| in   | cmd.sup_params.get_report | null       | Requests information about known parameters.                                                                 |
+| out  | evt.sup_params.report     | object     | Reports known parameters of the device, contains array of [`sup_parameter`](#definitions) objects.           |
+| in   | cmd.param.set             | object     | Sets parameter, see [`parameter_value`](#definitions).                                                       |
+| in   | cmd.param.get_report      | str_array  | Requests report with currently set values for specified parameters.                                          |
+| out  | evt.param.report          | object     | Reports current value for specified parameters, contains array of [`parameter_value`](#definitions) objects. |
+
+
+## Service properties
+
+| Name      | Type      | Example | Description                                                |
+|-----------|-----------|---------|------------------------------------------------------------|
+| sup_sizes | int_array | `[1,2]` | Optional. Defines supported sizes of the parameter values. |
+
+## Definitions
+
+* `sup_parameter` is an object of the following structure:
+
+| Field         | Type   | Example                                                                  | Description                                                                    |
+|---------------|--------|--------------------------------------------------------------------------|--------------------------------------------------------------------------------|
+| parameter_id  | string | `"1"`                                                                    | ID of the parameter.                                                           |
+| name          | string | `"Sensitivity of the PR sensor"`                                         | Name of the parameter.                                                         |
+| description   | string | `""`                                                                     | Description of the parameter.                                                  |
+| widget_type   | string | `"select"`                                                               | Format of the parameter, one of [`widget_type`](#definitions).                 |
+| value_type    | string | `"int"`                                                                  | Type of the value.                                                             |
```

Comment:
![Screenshot 2023-01-17 at 13 26 07](https://user-images.githubusercontent.com/24494656/212899970-f61233bf-f0bb-4f95-9300-4c1f4a12c40c.png)
do we need it to be duplicated?